<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="MessageHistory">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>781</width>
      <height>281</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;No message recived&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MessageText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="MessageHelp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>104</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Message</string>
    </property>
   </widget>
   <widget class="QLabel" name="GetterHelp">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>335</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Getter (nothing to send ALL)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="GetterText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SendButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QLabel" name="LoginHelp">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>67</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LoginText">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="PasswordHelp">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>111</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordText">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="RegButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>310</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>REGIESTER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LogButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>310</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LOGON</string>
    </property>
   </widget>
   <widget class="QLabel" name="PasswordHelp_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>136</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Importance</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RadioMin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Minimum</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RadioMed">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>101</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Medium</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RadioMax">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>250</y>
      <width>117</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Maximum</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameText">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="NameHelp">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>71</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
